README.md

```diff
@@ -103,12 +103,31 @@ This command shows the secret information including the URI. After completing th
 
     Open Program.cs file and update the placeholder <YourKeyVaultName> with the name of your Key Vault that you created in Step 3
 
-#### 7) Run the app
+#### 7) Set the environment variable for DefaultAzureCredential
+Follow one of the examples below depending on your operating system to create the environment variable. If using Windows close your open IDE or shell and restart it to be able to read the environment variable.
+
+Linux
```

Comment:
We should replace these with `az login` or `Connect-AzAccount` callouts. It's way simpler for customers than figuring out where to get credentials from.

Reply:
Step 1 already called `az login`. So only removing this details for setting environment variable.